cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -181,13 +181,13 @@ These metrics are available in a flowlet context:
      - Number of events emitted by the flowlet
    * - ``system.process.tuples.read``
      - Number of tuples read by the flowlet
-   * - ``system.store.bytes``
+   * - ``system.dataset.store.bytes``
      - Number of bytes written to datasets
-   * - ``system.store.ops``
+   * - ``system.dataset.store.ops``
      - Operations (writes and read) performed on datasets
-   * - ``system.store.reads``
+   * - ``system.dataset.store.reads``
```

Comment:
Is there an error in this doc now, because the only instance of ``system.store.reads`` is in the example under *Search for Metrics*:
```
   * - HTTP Method
     - ``POST '<base-url>/metrics/search?target=metric&tag=namespace:default&tag=app:HelloWorld&tag=service:Greeting'``
   * - Returns
     - | ``["system.dataset.store.ops","system.dataset.store.reads","system.requests.count",``
       | `` "system.response.successful.count","system.store.ops","system.store.reads",``
```
yet, this does not appear under the service context, nor anywhere else.